app/src/java/java/co/omise/android/example/CheckoutActivity.java

```diff
@@ -240,13 +240,16 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
             if (data.hasExtra(OmiseActivity.EXTRA_SOURCE_OBJECT)) {
                 Source source = data.getParcelableExtra(OmiseActivity.EXTRA_SOURCE_OBJECT);
                 snackbar.setText(source.getId()).show();
+                Log.d(TAG, "source: " + source.getId());
```

Comment:
We're logging on purpose here for the sake of the example app?

Reply:
Yes, just in case we want to use the token to create a charge.